docs/data/galexie/admin_guide/running.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Running
+sidebar_position: 40
+---
+
+# Running
+
+With the Docker image available and the configuration file set up, you're now ready to run Galexie and start exporting Stellar ledger data to the GCS bucket.
+
+## Command Line Usage
+
+### Append Command
+
+This is the primary way of running Galexie. The `append` command operates in two distinct modes:
+
+- In continuous/unbounded mode, it starts exporting from the specified start ledger and continuously exports new ledgers that appear on the network until the process is interrupted.
+- In fixed range mode, it exports the specified range of ledgers and exits when done.
+
+Syntax:
+
+```shell
+stellar-galexie append --start <start_ledger> [--end <end_ledger>] [--config-file <config_file>]
```

Comment:
Actually isn't galexie run with `galexie` not `stellar-galexie`?

Reply:
Yes, you're right—if built locally, it will default to `galexie`. I'm open to changing it to but the reason the for using `stellar-galexie` here was that when(?) we publish a deb package, the binary will likely be named `stellar-galexie` for consistency with the rest like stellar-core, stellar-horizon etc.

Reply:
Ahh okay. I'm good with leaving it as is